doc/enterprise/audit_log.rst

```diff
@@ -1,40 +1,253 @@
+    :language: yaml
+    :start-at: format:
+    :end-at: json
+    :dedent:
+
+..  _audit-log-spaces:
+
+Specify the spaces to be logged
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since :doc:`3.0.0 </release/3.0.0>`, :ref:`audit_log.spaces <configuration_reference_audit_spaces>` is used to specify
+a list of space names for which data operation events should be logged.
+
+In the configuration below, only the events from the ``bands`` space are logged:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log/config.yaml
+    :language: yaml
+    :start-at: spaces:
+    :end-at: bands ]
+    :dedent:
+
+..  _audit-log-extract:
+
+Specify the logging mode in DML events
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since :doc:`3.0.0 </release/3.0.0>`, it is possible to
+force the audit subsystem to log the primary key instead of a full tuple in DML operations.
+To do so, set the :ref:`audit_log.extract_key <configuration_reference_audit_extract_key>` option to ``true``.
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log/config.yaml
+    :language: yaml
+    :start-at: extract_key:
+    :end-at: true
+    :dedent:
+
+..  _audit-log-run-read:
+
+Reading audit logs
+------------------
+
+In the example, the logs are written to the ``audit_tarantool.log`` file.
+Let's create a space ``bands`` and check the logs in the file after the creation:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log/myapp.lua
+    :language: lua
+    :start-after: Create space
+    :end-before: -- Insert data
+    :dedent:
+
+The audit log entry for the ``space_create`` event might look as follows:
 
 ..  code-block:: json
 
     {
-        "time": "2022-04-07T13:39:36.046+0300",
-        "remote": "",
-        "session_type": "background",
-        "module": "tarantool",
-        "user": "admin",
-        "type": "user_create",
-        "tag": "",
-        "description": "Create user alice"
+      "time": "2024-01-24T11:43:21.566+0300",
+      "uuid": "26af0a7d-1052-490a-9946-e19eacc822c9",
+      "severity": "INFO",
+      "remote": "unix/:(socket)",
+      "session_type": "console",
+      "module": "tarantool",
+      "user": "admin",
+      "type": "space_create",
+      "tag": "",
+      "description": "Create space Bands"
     }
 
-It is up to each company to decide exactly what activities to audit and what actions to take.
-System administrators, security engineers, and people in charge of the company may want to
-audit different events for different reasons. Tarantool provides such an option for each of them.
+Then insert one tuple to space:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log/myapp.lua
+    :language: lua
+    :start-after: load_data()
+    :end-before: box.space.bands:insert { 2
+    :dedent:
+
+As the ``extract_key`` option is set to ``true``, the audit system prints the primary key instead of the full tuple:
```

Comment:
> As the ``extract_key`` option is set to ``true``

This was many sentences ago and a user might forget this fact. So, I'd change to `If the ``extract_key`` option ...`